### PR TITLE
fix: Fixed the design of 'Tab List'

### LINKED ISSUE #1486: Fix design of 'Tab list' in 'People Show', 'Company Show' and 'Tasks'
###  Scope:
- ShowPageRightContainer component in both the people show and company show.
- TopBar in 'Tasks' page

# 'People' and 'Company' show

### Current design

gap between elements within the StyledTabListContainer is 4px

Each tab has a padding of 8px'

The 

Image

### Expected design

Set the gap between elements within the StyledTabListContainer to be 8px.

Modify the styling for individual tabs by setting the padding for the left and right sides to 0.

Image

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=9901-34423&mode=design&t=aiEVp36gXiI1JZeu-4

### Current design
The border is inside the container of the Tab, not on the gray line

Image

### Expected design
Border should be on the gray line

Image

# 'Tasks' page

### Current design

Each tab has a padding of 8px'

Image

### Expected design

Modify the styling for individual tabs by setting the padding for the left and right sides to 0.
Add gap of 8px and margin-left 8px on TabList container.

Image

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=5910-71591&mode=design&t=3YJj0dTADKmD1LhV-4

# Click effect on all TabList

Currently, there is no response when clicking on a tab, but we would like the background to change from 'tertiary' to 'quaternary' while the tab is being pressed.

https://github.com/twentyhq/twenty/assets/102394218/e3f318a7-805b-4476-afe8-7d4836f6e6fb

### PR DESCRIPTION
**Pull request by zvolcsey**
Hi! I fixed the design of 'Tab list'. I added 8px gap and changed the padding to 0.

Closes #1486

### CODE REVIEW

**Comment by lucasbordeau** [comment]
Hi @zvolcsey thank you very much for your first contribution ! 

Could you provide a screenshot of your working solution so I can review more rapidly ?

**Comment by lucasbordeau** [comment]
Also please feel free to join our Discord server with other contributors and where we are reactive every day : https://discord.gg/rYvMveY2

**Comment by zvolcsey** [comment]
Hi @lucasbordeau! I made some screenshoots of my solution. I changed the gap value from 1 (4px) to 2 (8px) and changed the padding-left and padding-right from 1 to 0.

code
old:
old
new:
new

File: front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx [hunk 1]
```diff
@@ -32,10 +32,10 @@ const StyledTabListContainer = styled.div`
   border-bottom: ${({ theme }) => `1px solid ${theme.border.color.light}`};
   box-sizing: border-box;
   display: flex;
-  gap: ${({ theme }) => theme.spacing(1)};
+  gap: ${({ theme }) => theme.spacing(2)};
   height: 40px;
-  padding-left: ${({ theme }) => theme.spacing(1)};
-  padding-right: ${({ theme }) => theme.spacing(1)};
+  padding-left: ${({ theme }) => theme.spacing(0)};
```
Comment (lucasbordeau) [inline front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx:37]: I think you can go with `padding: 0` or remove the padding if there's no inherited padding maybe ?

**Comment by zvolcsey** [comment]
Hi @lucasbordeau! There are updates in the issue. In the latest commit, I made the following improvements:

`ShowPageRightContainer.tsx`
- remove the padding
- I changed the `align-items` from `end` to `center`. => The bottom border is on the gray line.
old:
tablist-old
new:
tablist-new

`Task.tsx`
- I added `gap` of 8px and `margin-left` 8px.

`TopBar.tsx`
- remove padding
old:
task-old
new:
task-new

`Tab.tsx`
- The background change from `tertiary` to `quaternary` while the tab is being pressed.
click

**Review by charlesBochet — APPROVED** [review]
Thank you @zvolcsey. I'm merging it and I'll complete tests in a follow up PR

**Pull request merged**